### PR TITLE
add 'set to false missing values' in checkCFG

### PR DESCRIPTION
**Pull request by marcobarilari**
quick and dirty solution, I am sure more will come up so we can edit in a better way later

### CODE REVIEW

**Comment by CerenB** [comment]
I think this PR is conflicting what Remi did in the `createFilename.m`
could it be that Marco did not update with that change?

**Comment by marcobarilari** [comment]
yes, I guess your master was not updated, now it is ok ;)

**Comment by marcobarilari** [comment]
> yes, I guess your master was not updated, now it is ok ;)

btw, kept the code that is in my repo

File: createFilename.m [hunk 1]
```diff
@@ -92,47 +92,38 @@
...
 
 switch modality
-    
+
     case 'beh'
-        
+
         expParameters.fileName.events = ...
             [expParameters.fileName.base, runSuffix, '_events_date-' expParameters.date '.tsv'];
-
```
Comment (CerenB) [inline createFilename.m:108]: still your PR does not include Remi's last changes. I think something is missing there, you may need to pull from cpp-bids?

**Review by CerenB — CHANGES_REQUESTED** [review]

File: createFilename.m [hunk 2]
```diff
@@ -92,47 +92,38 @@
...
             expParameters.dirSuffix, expParameters.recSuffix, ...
             runSuffix, expParameters.echoSuffix, ...
             '_events_date-' expParameters.date '.tsv'];
-        
-        expParameters.fileName.stim = ...
-            [expParameters.fileName.base, ...
-            expParameters.acqSuffix, expParameters.ceSuffix, ...
-            expParameters.dirSuffix, expParameters.recSuffix, ...
-            runSuffix, expParameters.echoSuffix, ...
-            '_stim_date-' expParameters.date '.tsv'];
```
Comment (CerenB) [inline createFilename.m:126]: there we go! finally I made it more clear what's missing from Remi's latest changes.

**Review by CerenB — CHANGES_REQUESTED** [review]

File: createFilename.m (hunk 1 above)
Comment (marcobarilari) [reply createFilename.m:108]: I am up to date on git kraken, which 'last changes?'

File: createFilename.m (hunk 2 above)
Comment (marcobarilari) [reply createFilename.m:126]: I think I am lost :) What happened?

Comment (marcobarilari) [reply createFilename.m:126]: what happened?

**Comment by marcobarilari** [comment]
so, what should I do?

File: createFilename.m (hunk 2 above)
Comment (CerenB) [reply createFilename.m:126]: The red ones are somehow doesnt exist in your branch. And these are the ones I'll need for my behavioral experiment that Remi implemented just an hour ago. 
Somehow your branch doesn't carry the changes from Remi and I do not want to lose those changes. Not sure how did that happen.

Comment (marcobarilari) [reply createFilename.m:126]: ok, I have no idea. Probably some conflict while working in different branches somehow not all at the same stage. I propose that I delete my pr so we don't lose Remi's job. Is it ok?

**Pull request closed without merging**